browser-test/src/api_docs.test.ts

```diff
@@ -9,20 +9,64 @@ import {BASE_URL} from './support/config'
 describe('Viewing API docs', () => {
   const ctx = createTestContext()
 
-  it('Views API docs', async () => {
-    const {page, adminPrograms} = ctx
+  beforeEach(async () => {
+    const {page} = ctx
+    await dropTables(page);
     await seedPrograms(page)
 
+  })
+
+  it('Views active API docs', async () => {
+    const {page, adminPrograms} = ctx
+
+    await page.goto(BASE_URL)
+    await loginAsAdmin(page)
+
+    await adminPrograms.publishAllDrafts()
+    await page.click('text=API docs')
+
+    await validateScreenshot(page, 'comprehensive-program-active-version')
+
+    await page.selectOption('#select-slug', {value: 'minimal-sample-program'})
+    await validateScreenshot(page, 'minimal-program-active-version')
```

Comment:
**suggestion (blocking):** in cases where the test manipulates the UI and then does a screenshot assertion, please insert an assertion on the updated text content of the page before doing the screenshot validation. This avoids a race condition where if the server is too slow to update the screenshot will be of the pre-updated UI.